pandas/core/base.py

```diff
@@ -7,8 +7,10 @@
 from pandas.core import common as com
 import pandas.core.nanops as nanops
 import pandas.lib as lib
+from pandas.compat.func_defaults import TRANSPOSE_DEFAULTS
```

Comment:
Perhaps my brain is a little burnt from working on this PR, but I don't quite understand what you're saying.  What is the purpose of `numpy_compat.transpose`?